components/WorkoutForm.js

```diff
@@ -0,0 +1,262 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { exercises } from "@/lib/exercises";
+import findExerciseById from "@/utils/findExerciseById";
+
+// Styled Components
+const Header = styled.header`
+  text-align: center;
+  margin-bottom: 1rem;
```

Comment:
I would suggest using px instead of rem for margins and paddings for more accessibility